example/server/index.ts

```diff
@@ -972,6 +974,113 @@ app.post('/detach-payment-method', async (req, res) => {
   });
 });
 
+app.get('/on-behalf-of-accounts', async (_, res) => {
```

Comment:
I don't think we should be updating this server, we should just remove it, and prefer all our server code lives in sandbox-apps and we run that server locally, otherwise we have two servers with two different implementations that are not sync'd.